src/service/actionssdk/conv.ts

```diff
@@ -98,9 +98,11 @@ export class ActionsSdkConversation<
       expectUserResponse,
       userStorage,
       expectedIntent,
+      noInputPrompts,
     } = this.response()
     const inputPrompt: Api.GoogleActionsV2InputPrompt = {
       richInitialPrompt: richResponse,
+      noInputPrompts: noInputPrompts,
```

Comment:
Can you change this to use [Object shorthand property notation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer#New_notations_in_ECMAScript_2015)?

It reduces code duplication (since key and value is same variable).

Reply:
I agree that.